sonar-helm-for-iac/converters/data-loading_test.go

```diff
@@ -8,39 +8,36 @@ import (
 
 func Test_store_multiple_sources_in_memory(t *testing.T) {
 	templateSources := &TemplateSources{
-		Name:        "test-project/templates/a.yaml",
-		RawTemplate: "apiVersion: v1",
-		fileNameToFileContent: map[string]string{
-			"values.yaml": "foo: bar",
-			"Chart.yaml":  "name: test-project",
-		},
+		Name: "test-project/templates/a.yaml",
+		files: filesFromStrings(map[string]string{
+			"test-project/templates/a.yaml": "apiVersion: v1",
```

Comment:
I'm not sure if it is related to this PR but all paths should be "normalized" to Chart root directory, the `templates/a.yaml` in this case. This will be important for "include" function

Reply:
Ok, I see. I included also the chart name here, because it should be included in `Template.Name`. However, I'll check and update the logic not to keep it in the map key